### ISSUE TITLE
ActivityPub / ActivityStreams

### ISSUE THREAD

**Issue by carlbarrdahl** [pfrazee/ctzn#1]
Hi! 👋 
Thank you for an interesting project! 

Would it be possible to align this with the ActivityPub and ActivityStreams standards? Doing this could potentially make other instance interoperable I believe?
https://www.w3.org/TR/activitypub/
https://www.w3.org/TR/activitystreams-core/

In addition, extending the Objects (https://www.w3.org/TR/activitystreams-core/#object) with schemas from schema.org could allow for a wide variety of data to be published. For example:
https://schema.org/Event
https://schema.org/Offer
https://schema.org/BlogPosting
(+ lots more)

Here's a repo of schema.org schemas converted to json schemas:
https://github.com/charlestati/schema-org-json-schemas

Perhaps this could enable querying on posts in a geolocation? Or for example podcast episodes (https://schema.org/PodcastEpisode).

I guess the idea would be like a twitter feed only with different types of posts to aid in discovery and exploration of new feeds.

Would love to hear your thoughts on this.

**Comment by pfrazee** [comment]
Hi Carl!

So among the nodejs community this idea of “mad science” projects grew in reaction to people constantly telling us “you can’t do THAT in JS!” We developed this response which was, “hey it’s about pursuing a spark in my head not building according to other people’s spark. It’s mad science, not normal science!” It’s kind of a shield from external opinion that acknowledges that, yeah, it might be a bad idea in the end but let’s play it out. 

This is one of those projects. The goal here is to pursue a novel architecture and see if it results in something better or worse than prior art. Worse? Then we have a useful null result. Better? Great!

That said, if you have a spark to use the ideas in ctzn to interact with APub, do it! In the name of mad science! I think that could be a great idea, it’s just not my spark.

**Comment by carlbarrdahl** [comment]
Hey, thanks for your reply.
I get it and I love that appoach! Apologies if the message came across as a request, I intended it more as something to discuss. 
We all have areas and ideas that interest us and huge respect to you for the work you do.

**Comment by pfrazee** [comment]
No not at all, totally cool you opened the issue, I just figured this will be a common question so I gave my "big answer." Cheers!

**Comment by danrobi11** [comment]
> like a twitter feed only 

Yes and only that. We dont really need "with different types of posts to aid in discovery" Thatss looking for trouble. (lobby's etc)
There could be a search engine. You could do a search for carlbarrdahl. If the ctzn team does not approve carlbarrdahl content for x reason they could block carlbarrdahl discovery at the search engine level. However, carlbarrdahl can always be reachable from direct link. That's how lbry work actually. It seems to work in order to stay out of trouble with lobby's, gov's etc.

my 2 cents
